CMakeLists.txt

```diff
@@ -396,6 +397,16 @@ install(TARGETS KVEngine Launcher
 install(FILES ${CMAKE_BINARY_DIR}/metadata.json
         DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/${EXEC_DIR}/)
 
+IF(WIN32)
+    SET(KVENGINE_EXEC ${CMAKE_CURRENT_SOURCE_DIR}/${EXEC_DIR}KVEngine.exe)
```

Comment:
The executable does not exist during cmake configuration stage (because compilation will be _after_ cmake configuration)